src/lib.rs

```diff
@@ -649,6 +648,21 @@ impl dyn TestStatus {
     }
 }
 
+fn retry_with_backoff<T>(mut f: impl FnMut() -> io::Result<T>) -> io::Result<T> {
+    let mut res = f();
+    for i in 0..10 {
+        if let Err(err) = &res {
+            if io::ErrorKind::WouldBlock == err.kind() || i < 5 {
```

Comment:
Do you really want to retry on arbitrary failures? That sounds wrong to me.

Reply:
Well... We also get arbitrary permission denied failures on process spawn, and I have no clue why